packages/web/components/route-prefetcher.tsx

```
@@ -0,0 +1,96 @@
+        }
+        break
+
+      case 'recent':
+        // Prefetch recently updated notes
+        const sortedByDate = [...notes]
+          .filter((note) => !note.isFolder && note.id !== activeNoteId)
+          .sort(
+            (a, b) =>
+              new Date(b.updatedAt).getTime() - new Date(a.updatedAt).getTime()
+          )
+          .slice(0, maxPrefetch)
```

Biome (2.1.2)

[error] Other switch clauses can erroneously access this declaration.
Wrap the declaration in a block to restrict its access to the switch clause.

The declaration is defined in this switch clause:

Safe fix: Wrap the declaration in a block.


(lint/correctness/noSwitchDeclarations)